### ISSUE TITLE
IT INSTANTLY DELETES THE ZIP

### ISSUE THREAD

**Issue by RECKLEXZ** [EZFNDEV/FNPrivateServer#6]
I download it view in map and then says removed or deleted

**Comment by RECKLEXZ** [comment]
HELP MAH PLEASEeeee

**Comment by ghost** [comment]
Sounds like you have an anti-virus giving you false positives, disable it.

**Comment by Jawschamp** [comment]
Yeah if it detects a Virus most anti-virus software's delete automatically.

**Comment by ghost** [comment]
> 
> 
> Yeah if it detects a Virus most anti-virus software's delete automatically.

False positive. This gets picked up as a "Game Hack" by windows because it injects into the client, not a virus.
